pkg/testutils/release/releases_test.go

```diff
@@ -61,8 +68,8 @@ func TestLatestAndRandomPredecessor(t *testing.T) {
 	}{
 		{
 			name:        "non-existent release",
-			v:           "v23.2.3",
-			expectedErr: `no release information for "v23.2.3" ("23.2" series)`,
+			v:           "v50.8.3",
```

Comment:
Nice, we'll need to update this in 2050!

Reply:
Hehe, not really, this test is reading from `testReleaseData`, not from the real YAML file. I added a 23.2 entry in the test data to test the changes to support pre-releases, that's why I needed a new non-existent version in this test.